auth/src/lib.rs

```diff
@@ -29,7 +29,7 @@ pub async fn start(pool: SqlitePool, args: StartArgs) -> io::Result<()> {
     let router = api::ApiBuilder::new()
         .with_sqlite_pool(pool)
         .with_sessions()
-        .into_router();
+        .into_router(stripe::Client::new(args.stripe_secret_key));
```

Comment:
Should we rather add the stripe client with a builder method?

Reply:
I don't have a strong preference for this. I haven't used a builder method for it because I wanted to pass it directly to the scope which will use it, which will attach it to the UserManager. Didn't feel the need to store the client in both ApiBuilder and UserManager. Either way should be fine though, lmk if you have a strong preference for having a builder method for it.

Reply:
I agree. Will improve this a bit on the lines you suggested.

Reply:
Done.